mod/events.php

```diff
@@ -415,16 +415,16 @@ function events_content(App $a) {
 	}
 
 	// Passed parameters overrides anything found in the DB
-	if($mode === 'edit' || $mode === 'new') {
-		if(!x($orig_event)) $orig_event = array();
+	if ($mode === 'edit' || $mode === 'new') {
+		if (!x($orig_event)) $orig_event = array();
 		// In case of an error the browser is redirected back here, with these parameters filled in with the previous values
-		if(x($_REQUEST,'nofinish')) $orig_event['nofinish'] = $_REQUEST['nofinish'];
-		if(x($_REQUEST,'adjust')) $orig_event['adjust'] = $_REQUEST['adjust'];
-		if(x($_REQUEST,'summary')) $orig_event['summary'] = $_REQUEST['summary'];
-		if(x($_REQUEST,'description')) $orig_event['description'] = $_REQUEST['description'];
-		if(x($_REQUEST,'location')) $orig_event['location'] = $_REQUEST['location'];
-		if(x($_REQUEST,'start')) $orig_event['start'] = $_REQUEST['start'];
-		if(x($_REQUEST,'finish')) $orig_event['finish'] = $_REQUEST['finish'];
+		if (x($_REQUEST,'nofinish')) $orig_event['nofinish'] = $_REQUEST['nofinish'];
+		if (x($_REQUEST,'adjust')) $orig_event['adjust'] = $_REQUEST['adjust'];
+		if (x($_REQUEST,'summary')) $orig_event['summary'] = $_REQUEST['summary'];
+		if (x($_REQUEST,'description')) $orig_event['description'] = $_REQUEST['description'];
+		if (x($_REQUEST,'location')) $orig_event['location'] = $_REQUEST['location'];
+		if (x($_REQUEST,'start')) $orig_event['start'] = $_REQUEST['start'];
+		if (x($_REQUEST,'finish')) $orig_event['finish'] = $_REQUEST['finish'];
```

Comment:
I'd rather there was an `event` key in `$_REQUEST` that held all those parameters.